src/ppx/ppx_lwt.ml

```diff
@@ -240,37 +235,37 @@ let lwt_expression mapper exp attributes ext_loc =
   | Pexp_ifthenelse (cond, e1, e2) ->
     let e2 =
       match e2 with
-      | None -> [%expr Lwt.return_unit] [@metaloc !default_loc]
+      | None -> let loc = !default_loc in [%expr Lwt.return_unit]
       | Some e -> e
     in
     let cases =
+      let loc = !default_loc in
       [
-        Exp.case ([%pat? true] [@metaloc !default_loc]) e1 ;
-        Exp.case ([%pat? false] [@metaloc !default_loc]) e2 ;
+        Exp.case [%pat? true] e1 ;
+        Exp.case [%pat? false] e2 ;
       ]
     in
     let new_exp =
+      let loc = !default_loc in
       [%expr Lwt.bind [%e cond] [%e Exp.function_ cases]]
-        [@metaloc !default_loc]
     in
-    Some (mapper.expr mapper { new_exp with pexp_attributes })
+    Some (mapper#expression { new_exp with pexp_attributes })
 
   | _ ->
     None
 
 let warned = ref false
 
-let mapper =
-  { default_mapper with
+class mapper = object (self)
+  inherit Ast_traverse.map as super
 
-    structure = begin fun mapper structure ->
-      if !warned then
-        default_mapper.structure mapper structure
+  method! structure = begin fun structure ->
+      if !warned then super#structure structure
```

Comment:
Please reduce formatting changes like here, where the change on old line 268 also wrongly appears to affect old line 267. This applies to many of the other diff hunks also, parts of which have changed only in indentation or splitting of tokens across lines. The formatting in this file is not great, but we should change it in separate work because this change is a bit scary and keeping it focused will be much easier for review, and for debuging later, if necessary, if this ends up as someone's `git bisect` endpoint :)

Reply:
Yes I understand it very well, sorry about that. I hope I fixed all superfluous changes.